test/test_torch.py

```diff
@@ -2500,6 +2503,10 @@ def test_serialization(self):
         b = [a[i % 2] for i in range(4)]
         b += [a[0].storage()]
         b += [a[0].storage()[1:4]]
+        b += [torch.range(1, 10).int()]
+        t1 = torch.FloatTensor().set_(a[0].storage()[1:4], 0, (3,), (1,))
+        t2 = torch.FloatTensor().set_(a[0].storage()[1:4], 0, (3,), (1,))
+        b += [(t1.storage(), t1.storage(), t2.storage())]
```

Comment:
Wasn't there a bug with storages that have `offset == 0`? Can you please add that to the test case?